elasticgraph-indexer_autoscaler_lambda/lib/elastic_graph/indexer_autoscaler_lambda/lambda_function.rb

```diff
@@ -26,7 +26,9 @@ def handle_request(event:, context:)
           min_cpu_target: event.fetch("min_cpu_target"),
           max_cpu_target: event.fetch("max_cpu_target"),
           maximum_concurrency: event.fetch("maximum_concurrency"),
-          indexer_function_name: event.fetch("indexer_function_name")
+          required_free_storage_in_mb: event.fetch("min_free_storage_in_mb"),
```

Comment:
/nit it feels inconsistent to call it `required_free_storage_in_mb` internally but call it `min_free_storage_in_mb` in the event payload.  Can you call it `required_free_storage_in_mb` in both spots?  Every other `event` key gets passed through as-is instead of getting renamed...

Reply:
ah yeah good point, that should match